scripts/prepare-fulfillment-service.sh

```
@@ -22,8 +20,18 @@ FULFILLMENT_API_URL=https://$(oc get route -n ${INSTALLER_NAMESPACE} fulfillment
+    # Trigger a one-time publish-templates AAP job
+    AAP_ROUTE_HOST=$(oc get routes -n "${INSTALLER_NAMESPACE}" --no-headers osac-aap -o jsonpath='{.spec.host}')
+    AAP_URL="https://${AAP_ROUTE_HOST}"
+    AAP_TOKEN=$(oc get secret osac-aap-api-token -n "${INSTALLER_NAMESPACE}" -o jsonpath='{.data.token}' | base64 -d)
+    JT_ID=$(curl -sk -H "Authorization: Bearer ${AAP_TOKEN}" \
+        "${AAP_URL}/api/controller/v2/job_templates/?name=osac-publish-templates" | jq -r '.results[0].id')
+    echo "Launching publish-templates AAP job (template ID: ${JT_ID})..."
+    curl -sk -X POST -H "Authorization: Bearer ${AAP_TOKEN}" -H "Content-Type: application/json" \
+        "${AAP_URL}/api/controller/v2/job_templates/${JT_ID}/launch/"
+
+    echo "Waiting for computeinstancetemplate ${INSTALLER_VM_TEMPLATE} to be published..."
     retry_until 1200 5 '[[ -n "$(osac get computeinstancetemplate -o json | jq -r --arg tpl ${INSTALLER_VM_TEMPLATE} '"'"'select(.id == $tpl)'"'"' 2> /dev/null)" ]]' || {
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)